deployments/automated-previews.mdx

```diff
@@ -207,3 +207,51 @@ destroy_preview:
     - if: $CI_MERGE_REQUEST_ID
       when: manual
 ```
+
+## BitBucket Pipelines
+
+### Creating and cleaning up preview environments
+
+This job can be pasted into your `bitbucket-pipelines.yml` at the root of your repository. Be sure to add repository variables for the ones mentioned in the warning below.
+
+On pull request creation, an Architect environment will be created. The component specified in the repository will be registered with the Architect Cloud and
+deployed to the environment. The pipeline provides the option to remove the deployed component and its environment from the Architect Cloud manually,
+which should be done once the pull request is merged or closed.
+
+<Warning>
+  This example assumes that the repo has `ARCHITECT_EMAIL`, `ARCHITECT_PASSWORD`, `ARCHITECT_ACCOUNT`, and `ARCHITECT_CLUSTER` set as CI/CD variables.
+</Warning>
+
+<Warning>
+  `ARCHITECT_PASSWORD` must be a [personal access
+  token](https://cloud.architect.io/login).
```

Comment:
We'll need to add somewhere in the docs about PATs. 

For the time being, we should probably expand this to chat about PATs so the user at least knows where they're located at and how to update them

Reply:
I'll just add a section about that in another PR now so that it can be included here

Reply:
https://gitlab.com/architect-io/product/-/issues/281

Reply:
Updated to include link to PATs